bedrock/careers/management/commands/sync_greenhouse.py

```diff
@@ -78,7 +73,6 @@ def __iter__(self):
 cleaner = bleach.sanitizer.Cleaner(
     tags=ALLOWED_TAGS,
     attributes=ALLOWED_ATTRS,
-    css_sanitizer=CSSSanitizer(allowed_css_properties=ALLOWED_STYLES),
```

Comment:
This is good. Did we want to consider removing `class` attributes as well? Since, the ones I saw reference classes that don't exist on bedrock.

Reply:
Yea they have the `"gmail_default"` there on the container… but it's at least to grab the content in DOM if needed, so I didn't bother to xcheck all the content it's not actually breaking anything… (will grep/sed again and check what classes are there at all)

Reply:
So the most common classes are: `external-link`, `content-pay`, `pay-input`, `description`, `title`, `pay-range`, `divider`, etc. … so I'd probably keep them in, in case they'd need to be targeted at some point?

Reply:
Thanks for double-checking.